RecoverVm/RecoverVM.ps1

```diff
@@ -0,0 +1,32 @@
+﻿Param(
+    [Parameter(Mandatory=$true)][string]$ServiceName ,
+    [Parameter(Mandatory=$true)][string]$VMName 
+)
+
+$Sub = Get-AzureSubscription -Current
+if ( ! $Sub ) 
+{
+    Write-Output "no current subscription set - please run add-azureaccount + select-azuresubscription -current first for the subscription containing your target vm!"
```

Comment:
The recovery modules focus on recovery and assume subscriptions are setup to reduce the # of required parameters. We can of course add an extra layer on top that does subscription plumbing